src/browser/atom-application.coffee

```diff
@@ -411,6 +415,41 @@ class AtomApplication
         console.log("Killing process #{pid} failed: #{error.code ? error.message}")
     delete @pidsToOpenWindows[pid]
 
+  saveState: ->
+    states = []
+    for window in @windows
+      if loadSettings = window.getLoadSettings()
+        states.push(_.pick(loadSettings,
+          'initialPaths'
+          'isSpec'
```

Comment:
It'd be unappealing to fire up Atom, only to have your laptop fan start to immediately spin because specs are running.